lighthouse-core/gather/gather-runner.js

```diff
@@ -86,13 +86,20 @@ class GatherRunner {
    * @return {!Promise}
    */
   static loadPage(driver, options) {
-    return driver.gotoURL(options.url, {
-      waitForLoad: true,
-      disableJavaScript: !!options.disableJavaScript,
-      flags: options.flags,
-    }).then(finalUrl => {
-      options.url = finalUrl;
-    });
+    const resetStorage = !options.flags.disableStorageReset;
+    const recordTrace = options.config.recordTrace;
+    const useThrottling = options.config.useThrottling;
+    return Promise.resolve()
```

Comment:
what do you think about moving this to `pass`? It seems more like setup for navigating (like `driver.beginDevtoolsLog` and `driver.beginTracing`) rather than just the act of navigating (putting it here also means going to `about:blank` also goes through these steps, FWIW)